### PR TITLE
Cleaning up the sphinx documentation as discussed at dev meeting

### PR DESCRIPTION
**Pull request by keeeto**
I want to close issue #21786. This branch is the beginning of work to clean up and clarify the documentation. The Sphinx documentation has been updated to:

* Include a section on training and tutorials
* Ensure that the documentation link at the top of each page returns to the top of the Sphinx docs page
* Add date stamps to pages
* Remove low resolution image from the front page

**To test:**

Build new documentation and make sure that:

* All top level links work
* Directory structure makes sense

**Release Notes** 

This does not need to be included in the release notes.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -146,6 +146,10 @@
 # If true, "(C) Copyright ..." is shown in the HTML footer. Default is True.
 html_show_copyright = False
 
+# Add the last updated information to the bottom of pages. Blank string implies
+# '%b %d, %Y' format.
+html_last_updated_fmt = ""
+
```
Comment (KarlPalmen) [inline docs/source/conf.py]: What is %b %d %Y format?
Note that Month, Day, Year format is not internationally acceptable. Instead the ISO YYYY-MM-DD format could be used.

**Comment by keeeto** [comment]
Thanks Karl, noted and amended as per your suggestion.

File: docs/source/conf.py (hunk 1 above)
Comment (KarlPalmen) [reply docs/source/conf.py]: :+1:

**Comment by KarlPalmen** [comment]
Actually having built the version before the date format change, I find "Feb 23, 2018" acceptable. It is "2\23\2018" etc. that are not acceptable.

**Review by KarlPalmen — APPROVED** [review]
This works OK. I approve with either the old or new date format, but if you go back to the old date format, a better comment including an example date is needed.

File: docs/source/index.rst [hunk 2]
```diff
@@ -7,10 +7,6 @@
 
 .. _contents:
 
-.. image:: images/Mantid_Logo_Transparent.png
```
Comment (martyngigg) [inline docs/source/index.rst]: :+1:

**Pull request merged**